SPECS/python-gevent/python-gevent.spec

```diff
@@ -1,16 +1,16 @@
 Summary:        Coroutine-based network library
 Name:           python-gevent
-Version:        1.3.6
-Release:        9%{?dist}
+Version:        21.1.2
```

Comment:
This seems to be a huge jump from `1.3.6`

I also recall a previous PR that was eventually not merged because of such a huge jump.
Reference: https://github.com/microsoft/CBL-Mariner/pull/6350

Did we try to build all the dependent packages as well ?

+1 for `3.0-dev`

/cc @jslobodzian @0xba1a

Reply:
The bug I linked has some more context on why we're looking for such a bump in version number. Per @frhuelsz's findings, it looks like the package never worked to begin with. I confirmed this by running a few simple functions via python CLI. So our goal became to find a gevent version that can be built with existing dependencies in Mariner 2, and I found this version to be the most recent one that works.

No package currently depends on gevent I believe. And for 3.0-dev, we already updated gevent to its latest version: https://github.com/microsoft/CBL-Mariner/pull/6849